dataprofiler/labelers/character_level_cnn_model.py

```diff
@@ -759,27 +759,21 @@ def predict(self, data, batch_size=32, show_confidences=False,
             allocation_index += num_samples_in_batch
 
         # Convert predictions, confidences to lists from numpy
-        predictions = predictions.tolist()
+        r_predictions = []
+        r_confidences = []
         if show_confidences:
-            confidences = confidences.tolist()
-        else:
-            confidences = None
-
-        # Trim array size to number of samples
-        if len(predictions) > allocation_index:
```

Comment:
so, we won't care about the allocation_index? Is that sufficient for memory allocation?

Reply:
The max memory usage didn't change between implementations. When we did "predictions.tolist()" it already converted everything to a list.